### DISCUSSION TITLE
Request for new release

### DISCUSSION THREAD

**Discussion by jacek-d** [zalopay-oss/jmeter-grpc-request#87] [General] [answered]
Hi All,

Thank you for building and sharing this plugin, it significantly simplifies testing of gRPC APIs.

There are many useful changes since version 1.1.1. I would appreciate if you could release version 1.2.0 with the changes in current master (after merging PR#84).

Kind regards,

Jacek

**Comment by huynhminhtan** [comment, marked as answer]
Yeah, please refer to the release [v1.2.0](https://github.com/zalopay-oss/jmeter-grpc-request/releases/tag/v1.2.0).

**Reply by jacek-d** [reply]
Thank you :)

**Reply by yl-yue** [reply]
This is good news, and I will be promoting this version in the open source community
